code/modules/mob/living/simple_animal/abnormality/waw/generalb.dm

```diff
@@ -0,0 +1,176 @@
+/mob/living/simple_animal/hostile/abnormality/general_b
+	name = "General Bee"
+	desc = "A bee humanoid creature."
+	icon = 'ModularTegustation/Teguicons/48x96.dmi'
+	icon_state = "generalbee"
+	icon_living = "generalbee"
+	faction = list("hostile")
+	speak_emote = list("buzzes")
+	pixel_x = -8
+	base_pixel_x = -8
+	threat_level = WAW_LEVEL
+	start_qliphoth = 1
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(0, 0, 55, 55, 60),
+						ABNORMALITY_WORK_INSIGHT = list(0, 0, 45, 45, 50),
+						ABNORMALITY_WORK_ATTACHMENT = 0,						//DO NOT FUCK THE BEEGIRL
+						ABNORMALITY_WORK_REPRESSION = list(0, 0, 40, 40, 40)
+						)
+	work_damage_amount = 10
+	work_damage_type = RED_DAMAGE
+	ego_list = list(
+		/datum/ego_datum/weapon/loyalty,
+		/datum/ego_datum/armor/loyalty
+		)
+	//She doesn't usually breach. However, when she does, she's practically an Aleph-level threat. She's also really slow, and should pack a punch.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.3, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 1)
```

Comment:
she should still probably be a little less resistant, even if it's just giving her a pale weakness.

Reply:
She breaches under extremely specific and rare circumstances, she should be extremely powerful.
She has to spawn both at the same time as queen bee, and NEARBY queen bee.

Reply:
Fair. Though I'd definitely like to have an eye kept on it while it's Test-Merged